lib/package-managers/npm.js

```diff
@@ -44,6 +44,30 @@ function parseJson(result, data) {
     return json;
 }
 
+/**
+ * Check if package author changed between current and next version.
+ * @param {string} packageName Name of the package
+ * @param {string} currentVersion
+ * @param {string} latestVersion
+ * @returns {Promise<boolean>} A promise that fullfills with boolean value.
+ */
+async function packageAuthorChanged(packageName, currentVersion, latestVersion) {
+    npmConfig.fullMetadata = true;
+
+    const result = await pacote.packument(packageName, npmConfig);
+    if (result.versions) {
+        let pkgVersions = Object.keys(result.versions);
+        const current = semver.maxSatisfying(pkgVersions, currentVersion);
```

Comment:
I think you want to use `currentVersion` as-is rather than getting `maxSatisfying`. If the owner changed on a patch version in between `currentVersion` and `current`, it would not detect the owner change.

Reply:
Thinking about the bug after removing the `maxSatisfying`, I tested the counterpart function (`minSatisfying`) and I think it does what I wanted for both **current** and **latest** version:

> Return the lowest version in the list that satisfies the range, or null if none of them do.

So if Mocha **current** is `^7.0.1`, the minimum version will be `7.0.1` whereas if **latest** is `8.0.0` or `^8.0.0`, it will return `8.0.0`. For cases where the version doesn't exist, i.e. **current** being `8.2.1`/ `^8.2.1` and **latest** being `9.0.0`, it will fail as `minSatisfying` will be unable to find a version to match these.

Reply:
One image is better than 1000 words.

![ncu-owner-check](https://user-images.githubusercontent.com/30560560/87855561-d3cddb80-c8de-11ea-94a6-f05df8ccc4bb.png)

Reply:
I can confirm that `currentVersion` is a range, not an exact version as I suggested in my initial comment.

I changed it to use `minSatisfying` on `currentVersion` and `maxSatisfying` on `upgradedVersion`. This ensures that the broadest change is covered in case the owner changed anywhere between the two.